### ISSUE TITLE
Rancher HA Management stack never stabilizes

### ISSUE THREAD

**Issue by LLParse** [rancher/rancher#4344]
Rancher Version: v1.0.1-rc2

Docker Version: 1.10.0

OS: Ubuntu 15.10

Steps to Reproduce: Create an external Mysql database. Create a rancher server node and connect it to the database. Generate an HA deployment script and copy it to current server and two more servers. Start the two new servers with the HA script, then shutdown the original server and re-run with the HA script.

Results: Under System HA environment, the management stack never successfully deploys. The go-machine-service and rancher-compose-executor services appear as Initializing, start, stop, and return to Initializing.

Expected: A healthy HA management stack

**Comment by LLParse** [comment]
Go-machine-service doesn't like the self-signed certificate:
```
time="2016-04-11T20:36:48Z" level=info msg="Listening on port: 80" 
time="2016-04-11T20:36:48Z" level=info msg="Setting log level" logLevel=info 
time="2016-04-11T20:36:48Z" level=info msg="Starting go-machine-service..." gitcommit=dc97268 
time="2016-04-11T20:36:48Z" level=info msg="Updating docker-machine-drivers from cattle." 
time="2016-04-11T20:36:48Z" level=error msg="Error from driver: Get https://104.197.169.123/v1: x509: cannot validate certificate for 104.197.169.123 because it doesn't contain any IP SANs" 
time="2016-04-11T20:36:48Z" level=fatal msg="Encourted an error while updating drivers." 
time="2016-04-11T20:36:48Z" level=fatal msg="exit status 1" 
```

**Comment by LLParse** [comment]
Fixed in v1.0.1-rc5
